src/components/Modals/NewCourse.vue

```diff
@@ -167,7 +169,7 @@ export default {
     },
 
     addCourse() {
-      if (this.$refs[`dropdown-${this.semesterID}`].value) this.$parent.addItem();
+      if (this.$refs[`dropdown-${this.semesterID}`].value) this.$parent.addItem(this.semesterID);
```

Comment:
add item doesn't take a parameter? Also, could you change this to emit instead of $parent?

Reply:
The addItem in OnboardingTransfer.vue needs the id of a course when adding transfer credit classes